### PR TITLE
Changes to CIME to make CICE6 the default component.

### PR DESCRIPTION
**Pull request by dabail10**
This changes the default sea ice component from CICE5 to CICE6. The only CIME changes are to replace "cice6" with "cice5" in three files.

Test suite: aux_cice (formerly aux_cice6)
Test baseline:
Test namelist changes:
Test status: Changing from CICE5 to CICE6 is answer changing for all active sea ice cases. They have been deemed to have the same climate.

Fixes [CIME Github issue #]

User interface changes?:

Update gh-pages html (Y/N)?:

### CODE REVIEW

**Review by jedwards4b — APPROVED** [review]

File: scripts/lib/CIME/XML/archive_base.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def get_all_hist_files(self, casename, model, from_dir, suffix="", ref_case=None
         # remove when component name is changed
         if model == "fv3gfs":
             model = "fv3"
-        if model == "cice6":
+        if model == "cice5":
```
Comment (jedwards4b) [inline scripts/lib/CIME/XML/archive_base.py:102]: the changes in config_files.xml are specific to model cesm.  So the changes in archive_base.py and case_st_archive.py will need to be applied only when cime_model == cesm.

**Review by jedwards4b — CHANGES_REQUESTED** [review]
We will need to add something so that the changes in python scripts only apply for cime_model == cesm.

File: scripts/lib/CIME/XML/archive_base.py (hunk 1 above)
Comment (dabail10) [reply scripts/lib/CIME/XML/archive_base.py:102]: Does E3SM have a cice6 or a cice5? Won't it ignore this anyway?

Comment (rljacob) [reply scripts/lib/CIME/XML/archive_base.py:102]: Could you just add an .or. statement?  If model = (cice5 or cice6)

Comment (rljacob) [reply scripts/lib/CIME/XML/archive_base.py:102]: Actually we don't have cice5 or cice6.  We just have "cice".  So this code probably doesn't affect E3SM.

Comment (jedwards4b) [reply scripts/lib/CIME/XML/archive_base.py:102]: oh good point - it only has cice so I think that it's okay.

**Review by jedwards4b — APPROVED** [review]

**Comment by jedwards4b** [comment]
False alarm - e3sm will never trigger these statements.

**Comment by mvertens** [comment]
@jedwards4b @rljacob - can this PR be merged?

**Comment by jedwards4b** [comment]
Yes

**Comment by rljacob** [comment]
yes

**Review by rljacob — APPROVED** [review]

**Pull request merged**